lib/dradis/plugins/settings.rb

```diff
@@ -44,6 +44,15 @@ def is_default?(key, value)
       value.to_s == @default_options[key.to_sym].to_s
     end
 
+    def engine_enable
+      enable = db_setting(:enable)
+      enable ? (enable == '1') : nil
```

Comment:
There's some weirdness around checking for the `1` string and it doesn't help with the readability of the code. What if we just save the strings: "enable" and "disable" to represent the enabled and disabled engines respectively?